tests/app/forms/test_questionnaire_form.py

```diff
@@ -1227,7 +1232,9 @@ def test_mandatory_mutually_exclusive_question_raises_error_when_not_answered(se
 
             self.assertEqual(
                 form.question_errors["mutually-exclusive-checkbox-question"],
-                "Enter an answer to continue",
+                format_message_with_title(
+                    error_messages["MANDATORY_CHECKBOX"], question_schema.get("title")
+                ),
```

Comment:
I think this test should test the standard error message rather than the checkbox one.